### PR TITLE
CLI: Fixed missing module “theme.js” error

### PR DESCRIPTION
**Pull request by kowalskidev**
Fixed issue: https://github.com/laurent22/joplin/issues/3092
As discussed with Laurent [here](https://github.com/laurent22/joplin/issues/3092#issuecomment-617110612), I moved `theme.js` and it's dependencies in `ReactNativeClient/lib` folder and refactored the code to change path. Now everything is working fine.

@laurent22  please let me know if any changes needed. 
@PackElend  Label me please. Thank you!

### CODE REVIEW

File: ReactNativeClient/lib/theme.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
-const Setting = require('lib/models/Setting.js');
+const Setting = require('./models/Setting.js');
```
Comment (laurent22) [inline ReactNativeClient/lib/theme.js:1]: Please don't change this

File: ReactNativeClient/lib/theme.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-	[Setting.THEME_SOLARIZED_DARK]: require('./gui/style/theme/solarizedDark'),
-	[Setting.THEME_NORD]: require('./gui/style/theme/nord'),
-	[Setting.THEME_ARITIM_DARK]: require('./gui/style/theme/aritimDark'),
+	[Setting.THEME_LIGHT]: require('./theme/light'),
+	[Setting.THEME_DARK]: require('./theme/dark'),
+	[Setting.THEME_DRACULA]: require('./theme/dracula'),
+	[Setting.THEME_SOLARIZED_LIGHT]: require('./theme/solarizedLight'),
+	[Setting.THEME_SOLARIZED_DARK]: require('./theme/solarizedDark'),
+	[Setting.THEME_NORD]: require('./theme/nord'),
+	[Setting.THEME_ARITIM_DARK]: require('./theme/aritimDark'),
```
Comment (laurent22) [inline ReactNativeClient/lib/theme.js:10]: Better to use lib/theme/....

File: ReactNativeClient/lib/theme.js (hunk 1 above)
Comment (kowalskidev) [reply ReactNativeClient/lib/theme.js:1]: Alright! I'll keep as it is.

File: ReactNativeClient/lib/theme.js (hunk 2 above)
Comment (kowalskidev) [reply ReactNativeClient/lib/theme.js:10]: Noted. Thanks.

**Comment by kowalskidev** [comment]
>  Did you check if exporting HTML from the CLI app works now?

Yes Yes I checked twice. It's working now.



> Does it also work with notes that contain Katex expressions and Mermaid?

I didn't checked with Katex and Mermaid expressions yet but give me a moment I'll check right now.

**Comment by kowalskidev** [comment]
> Does it also work with notes that contain Katex expressions and Mermaid?

I tested with katex and mermaid expression then I got this error:
`export: Error: ENOENT: no such file or directory, lstat '/home/zkesarani/Dev/joplin/CliClient/build/gui/note-viewer/pluginAssets` and I figured out the error is due to this line:
```javascript
const filePath = `${dirname(dirname(__dirname))}/gui/note-viewer/pluginAssets/${asset.name}`;
```
in `InteropService_Exporter_Html.js` it is correct for ElectronClient because of in-app rendering but that's not a case in CliClient so I think we can add validation like if `env==cli` then path will be lib/joplin-renderer/pluginAssets/ 

@laurent22 Any thoughts?

**Comment by kowalskidev** [comment]
> I tested with katex and mermaid expression then I got this error:
> `export: Error: ENOENT: no such file or directory, lstat '/home/zkesarani/Dev/joplin/CliClient/build/gui/note-viewer/pluginAssets`

@laurent22  Shall I open separate issue for this error or we can solve it in this PR?

**Comment by laurent22** [comment]
I think there's a bigger issue here which is to get the CLI to export HTML and PDF correctly. And doing that probably means further refactoring, in particular in relation to the plugin asset paths. I have no idea how to fix this at this point and it's relatively low priority so I suggest we close this issue for now.

**Comment by laurent22** [comment]
Thanks for your understanding @kowalskidev

**Pull request closed without merging**